scripts-circleci/README.md

```diff
@@ -11,3 +11,52 @@ Place the following snippet in your CircleCI config to call `package-helm` Githu
         # export VERSION_FILE=/customPath/version.txt
         curl -s "https://raw.githubusercontent.com/qlik-oss/ci-tools/master/scripts-circleci/github-package-helm-dispatch.sh" | bash
 ```
+
+`.github/workflows/package-helm.yaml`
+
+``` yaml
+name: Package Helm chart
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version used in Docker image tag and Helm chart'
+        required: true
+      commitsha:
+        description: 'Commit SHA'
+        required: false
+
+env:
+  VERSION: ${{ github.event.inputs.version }}
+  COMMITSHA: ${{ github.event.inputs.commitsha }}
+  # Constants
+  REGISTRY:
+  ARTIFACTORY_USERNAME:
+  ARTIFACTORY_PASSWORD:
+jobs:
+  package-chart:
+    name: Package Helm chart
+    runs-on: ubuntu-latest
+    steps:
+      # Prints Action envrionment information useful for debugging
+      - uses: qlik-oss/ci-tools/action-print-event-info@master
+
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.commitsha }}
```

Comment:
Here we checkout the commit that comes as an input (usually from circleci as CIRCLE_SHA1)